src/main/java/com/polyscripts/contactManagement/models/Contact.java

```diff
@@ -0,0 +1,25 @@
+package com.polyscripts.contactManagement.models;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.List;
+
+@Entity
+@Getter @Setter
```

Comment:
a simple @Data  annotation is sufficient here, you don't need to add getter and setter annotations